packages/live-preview-sdk/src/inspectorMode/utils.ts

```diff
@@ -1,40 +1,77 @@
 import { InspectorModeAttributes, InspectorModeDataAttributes } from './types';
 
+const isTaggedElement = (node: Node): boolean => {
+  // is Element?
```

Comment:
I'm not sure this comment helps we understand more what is going on

Reply:
At first I was using `node.nodeType === 1` before discovering the `Node.ELEMENT_NODE`

Reply:
Yeah, sure I will remove the comment. I was just explaining why it was there

Reply:
Thanks mate!